### PR TITLE
chore: tag OrderTaxesApplicatorInterface

### PR DESCRIPTION
**Pull request by Florian-Merle**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13                   |
| Bug fix?        | no                                                       |
| New feature?    | no                                                      |
| BC breaks?      | no                                                       |
| Related tickets | mentioned in #14609                      |
| License         | MIT                                                          |

### CODE REVIEW

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Hi @Florian-Merle!
Is this PR still a WIP? It seems to be ready 🤔.

P.S. I let myself rebase it against latest changes from 1.13 branch :).

**Comment by Florian-Merle** [comment]
Hi @jakubtobiasz ! Indeed it seems to be ready. Last time I worked on it the pipeline would fail but I did not take the time to search why 😅

File: src/Sylius/Bundle/CoreBundle/Resources/config/services/taxation.xml [hunk 1]
```diff
@@ -27,12 +27,15 @@
...
+            <tag name="sylius.taxation.item_units.applicator" />
         </service>
         <service id="sylius.taxation.order_items_taxes_applicator" class="Sylius\Component\Core\Taxation\Applicator\OrderItemsTaxesApplicator">
             <argument type="service" id="sylius.tax_calculator" />
             <argument type="service" id="sylius.factory.adjustment" />
             <argument type="service" id="sylius.integer_distributor" />
             <argument type="service" id="sylius.tax_rate_resolver" />
+            <tag name="sylius.taxation.items.applicator" />
         </service>
         <service id="sylius.taxation.order_item_units_taxes_applicator" class="Sylius\Component\Core\Taxation\Applicator\OrderItemUnitsTaxesApplicator">
```
Comment (Florian-Merle) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/taxation.xml:40]: @jakubtobiasz re-reading the code, I think this service is missing a tag

Comment (Florian-Merle) [reply src/Sylius/Bundle/CoreBundle/Resources/config/services/taxation.xml:40]: This should be good now :)

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Thanks! I'll wait for more approves/comments, and then I'll merge it :).

**Review by NoResponseMate — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Florian! :tada:

**Pull request merged**